### DISCUSSION TITLE
Re-attach RabbitMQ statefulset to the existing volumes

### DISCUSSION THREAD

**Discussion by Dragane** [rabbitmq/cluster-operator#1229] [Q&A]
I noticed that the old PVCs get deleted and new ones created when I re-install the RabbitMQ cluster. 

Is there any way to automatically re-attach the pods to the existing volumes, instead of deleting them?

**Comment by DanielePalaia** [comment]
Hi @Dragane I think that this discussion here: https://github.com/rabbitmq/cluster-operator/issues/855 can clarify to you this.
It was mainly a team decision to do this: "PVC gets garbage collected because the Operator sets owner reference in the created PVCs. The team made the decision to set owner references on the PVCs with the understanding that PV deletion is governed by it reclaim policy that can be configured by users."
Not sure if this is replying to your question, I'll move the issue to a discussion.
